include/sys/fs/zfs.h

```diff
@@ -528,6 +529,22 @@ typedef struct zpool_rewind_policy {
 #define	ZPOOL_CONFIG_DTL		"DTL"
 #define	ZPOOL_CONFIG_SCAN_STATS		"scan_stats"	/* not stored on disk */
 #define	ZPOOL_CONFIG_VDEV_STATS		"vdev_stats"	/* not stored on disk */
+#define	ZPOOL_CONFIG_VDEV_STATS_FLAGS	"vdev_stats_flags"	/* not stored */
+								/* on disk */
+
+/* Queue read/write stats */
+#define	ZPOOL_CONFIG_QUEUE_ACTIVE	"queue_active"
+/* Latency read/write histogram stats */
+#define	ZPOOL_CONFIG_TOT_R_LAT_HISTO	"tot_r_lat_histo"
+#define	ZPOOL_CONFIG_TOT_W_LAT_HISTO	"tot_w_lat_histo"
+#define	ZPOOL_CONFIG_DISK_R_LAT_HISTO	"disk_r_lat_histo"
+#define	ZPOOL_CONFIG_DISK_W_LAT_HISTO	"disk_w_lat_histo"
+#define	ZPOOL_CONFIG_SYNC_R_LAT_HISTO	"sync_r_lat_histo"
+#define	ZPOOL_CONFIG_SYNC_W_LAT_HISTO	"sync_w_lat_histo"
+#define	ZPOOL_CONFIG_ASYNC_R_LAT_HISTO	"async_r_lat_histo"
+#define	ZPOOL_CONFIG_ASYNC_W_LAT_HISTO	"async_w_lat_histo"
+#define	ZPOOL_CONFIG_SCRUB_LAT_HISTO	"scrub_histo"
```

Comment:
Perhaps the should be prefixed `ZPOOL_CONFIG_VDEV_*` for clarity.  What do you think.